crates/pipeline_manager/Cargo.toml

```
@@ -3,6 +3,11 @@ name = "dbsp_pipeline_manager"
 version = "0.1.0"
 edition = "2021"
 license = "MIT OR Apache-2.0"
+description = "Continuous streaming data analytics"
```

```suggestion
description = "Data pipeline manager for the Feldera continuous analytics platform."
```

which probably means that we should rename the crate to `feldera_pipeline_manager`.

@lalithsuresh , do you have an opinion on naming this crate?  We discussed splitting it into API gateway, runner service, and compiler service. But if they still live in the same crate, "pipeline manager" might still be a good name for it?

---

I was going to name it `pipeline-management` in my next PR when finalizing the splitting up.

---

I see.  Frankly, we might keep `manager` in that case, it's just going to be a distributed manager, not a centralized one as now. And I guess the full name of the crate (as it appears on crates.io) will be `feldera-pipeline-manager`. Can be another PR, but I think we should rename it before uploading to crates.io.

---

I thought we only wanted to upload the dbsp crate to crates.io? How are we expecting people to consume the pipeline management crate?

---

It's still a good idea to have metadata in all other crates in order.

---

My interpretation of this feedback:
* I'll update the description as @ryzhyk suggested.
* I'll leave the crate name alone.